modules/script/script-impl/src/main/java/com/enonic/xp/script/impl/util/JsObjectConverter.java

```diff
@@ -32,6 +33,11 @@ public Object toJs( final Object value )
             return toJs( (List) value );
         }
 
+        if ( value != null && value.getClass().isArray() )
+        {
+            return toJs( Arrays.asList( (Object[]) value ) );
```

Comment:
Not all arrays can be casted to Object[]

Reply:
done